### PR TITLE
PIM-543 adding support for labels

### PR DESCRIPTION
**Pull request by mfullmore**

### CODE REVIEW

File: lib/ImportAttributeLabel.js [hunk 1]
```diff
@@ -66,17 +66,16 @@ class ImportAttributeLabel extends ImportClass {
       tmpAttributeLabel[this.helper.namespace('Classification__c')] = node.classification
       tmpAttributeLabel[this.helper.namespace('Is_Localizable__c')] = node.localizable
       tmpAttributeLabel[this.helper.namespace('Is_Searchable__c')] = node.searchable
+      tmpAttributeLabel['Label__c'] = node.name
```
Comment (danielxji6) [inline lib/ImportAttributeLabel.js:69]: No need for `namespace`?

Comment (mfullmore) [reply lib/ImportAttributeLabel.js:69]: that is why we have reviews.

Comment (mfullmore) [reply lib/ImportAttributeLabel.js]: updated

**Review by danielxji6 — APPROVED** [review]
🔥

**Pull request merged**